src/com/google/common/css/compiler/gssfunctions/GssFunctions.java

```diff
@@ -370,7 +375,7 @@ public String getCallResultString(List<String> args)
           baseColorString, args.get(1), args.get(2), args.get(3));
     }
 
-    protected String addHsbToCssColor(
+    public static String addHsbToCssColor(
```

Comment:
I was just looking at `BaseBlendColors`. I wonder if you can't do something similar here by making all of the functions that use `AddHsbToCssColor.addHsbToCssColor` inherit from the same base class, including what is today `AddHsbToCssColor`. What do you think?